src/components/Restoration/ApprovalType.vue

```diff
@@ -3,6 +3,7 @@
     <div
       class="section-container"
       :class="{ 'invalid-section': invalidSection }"
+      :style="invalidSection ? {margin: '0 9px 0 0'} : {}"
```

Comment:
Paul, we usually don't add in-line styling. Would it be possible to add a conditional class for this maybe?

Reply:
Of course, I will change that right away.

Reply:
Unless you need exactly 9px of right margin, you can use a [Vuetify spacing helper class](https://v2.vuetifyjs.com/en/styles/spacing/) here. In this case, `mr-2`.

Reply:
By the way,  you can have multiple dynamic classes. Something like this:
```
:class="{
  'myClass1': myCondition1,
  'myClass2': myCondition2
}"
```

Reply:
So much easier, learning the power of Vue